ci-operator/step-registry/interop-tests/fusion-access/test-cnv-shared-storage/interop-tests-fusion-access-test-cnv-shared-storage-commands.sh

```
@@ -0,0 +1,263 @@
+    volumeMounts:
+    - name: shared-storage
+      mountPath: /shared-storage
+  volumes:
+  - name: shared-storage
+    persistentVolumeClaim:
+      claimName: test-simple-shared-pvc
+  restartPolicy: Never
+EOF
+    then
+      echo "  ✅ Test pod created"
+      
+      # Wait for pod to be running
+      echo "  ⏳ Waiting for test pod to be running..."
+      if oc wait pod test-shared-storage-pod -n "${TEST_NAMESPACE}" --for=condition=Ready --timeout=2m 2>/dev/null; then
+        echo "  ✅ Test pod is running"
+        
+        # Check pod logs
+        echo "  📊 Test pod logs:"
+        oc logs test-shared-storage-pod -n "${TEST_NAMESPACE}" --tail=10
+      else
+        echo "  ⚠️  Test pod not ready within timeout"
+        oc describe pod test-shared-storage-pod -n "${TEST_NAMESPACE}"
+      fi
+    else
+      echo "  ❌ Failed to create test pod"
+    fi
+  else
+    echo "  ⚠️  PVC not bound within timeout"
+    oc get pvc test-simple-shared-pvc -n "${TEST_NAMESPACE}" -o yaml
+  fi
+else
+  echo "  ❌ Failed to create simple PVC"
+fi
+
+# Check storage usage
+echo ""
+echo "📊 Storage Usage Summary:"
+echo "  📋 PVCs in test namespace:"
+oc get pvc -n "${TEST_NAMESPACE}" -o custom-columns="NAME:.metadata.name,STATUS:.status.phase,STORAGECLASS:.spec.storageClassName,CAPACITY:.status.capacity"
+
+echo "  📋 VMs in test namespace:"
+oc get vm -n "${TEST_NAMESPACE}" -o custom-columns="NAME:.metadata.name,STATUS:.status.printableStatus,AGE:.metadata.creationTimestamp"
+
+echo "  📋 Pods in test namespace:"
+oc get pods -n "${TEST_NAMESPACE}" -o custom-columns="NAME:.metadata.name,STATUS:.status.phase,AGE:.metadata.creationTimestamp"
+
+# Cleanup
+echo ""
+echo "🧹 Cleaning up test resources..."
+echo "  🗑️  Stopping VM..."
+oc patch vm test-shared-storage-vm -n "${TEST_NAMESPACE}" --type=merge -p '{"spec":{"running":false}}' 2>/dev/null || true
+
+echo "  🗑️  Deleting VM..."
+oc delete vm test-shared-storage-vm -n "${TEST_NAMESPACE}" 2>/dev/null || true
```

Why the STDERR is silenced and why we ignore the potential non-zero exit status of `oc`?

Do you want to make it idempotent? So in case of the resources to be deleted actually does not exist? If so, there is a better way to do it without blindly silence error message and ignore non-zero exit status of `oc`.

The `oc delete ... --ignore-not-found ...` will do that and we are not ignoring any other issue.

---

Resolved. CNV testing has been split into a separate PR (commit a18816507fb). This branch now focuses on EBS integration testing only. The test-cnv-shared-storage component was removed from PR scope.